src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -241,17 +244,17 @@ data class CommonCaseNoteFields(
   private fun equalTypes(nomis: CommonCaseNoteFields): Boolean = (type == nomis.type || (type == "APP" && nomis.type == "CNOTE" && subType == "OUTCOME"))
 
   private fun equalUsers(other: CommonCaseNoteFields): Boolean {
-    val equal = authorUsername == other.authorUsername || authorUsername == "OMS_OWNER"
+    val equal = other.nomisUsernames?.contains(dpsUsername) == true || dpsUsername == "OMS_OWNER"
```

Comment:
This is not ideal as it is asymetrical - though I am thinking of splitting this out to report what specific field does not match, when there is time